desktop/cmp/persistenceManager/PersistenceManager.ts

```diff
@@ -15,67 +15,67 @@ import {pluralize} from '@xh/hoist/utils/js';
 interface PersistenceManagerProps extends HoistProps<PersistenceManagerModel> {
     /** True to disable options for saving/managing items. */
     minimal?: boolean;
+    /** True (default) to render a save button alongside the primary menu button when dirty. */
+    enableTopLevelSaveButton?: boolean;
+    /** True to omit the default menu component. Should be used when creating custom app-specific component */
+    omitDefaultMenuComponent?: boolean;
```

Comment:
To discuss today - could make this `viewMenu: boolean | ViewMenuProps` and default it to `true` if we want something a bit more flexible. Then we could remove `enableTopLevelSaveButton` as a top-level prop here and make it part of `ViewMenuProps`